<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>465</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="runBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause/Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speedBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stepLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="speedLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dirBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Dir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statusBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="XlabelSpeed">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>XSpeed</string>
    </property>
   </widget>
   <widget class="QLabel" name="XlabelDir">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>XDir</string>
    </property>
   </widget>
   <widget class="QLabel" name="XlabelEnable">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>XEnable</string>
    </property>
   </widget>
   <widget class="QLabel" name="XlabelCur">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>XCurstep</string>
    </property>
   </widget>
   <widget class="QLabel" name="XlabelRemain">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>XRemain</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textExtract">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="YlabelRemain">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>YRemain</string>
    </property>
   </widget>
   <widget class="QLabel" name="YlabelCur">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>YCurStep</string>
    </property>
   </widget>
   <widget class="QLabel" name="YlabelSpeed">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>YSpeed</string>
    </property>
   </widget>
   <widget class="QLabel" name="YlabelEnable">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>YEnable</string>
    </property>
   </widget>
   <widget class="QLabel" name="YlabelDir">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>YDir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mNumLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dirLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="runDrawBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RUNSYS</string>
    </property>
   </widget>
   <widget class="QLabel" name="ZlabelEnable">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ZEnable</string>
    </property>
   </widget>
   <widget class="QLabel" name="ZlabelRatio">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ZRatio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZRatioBtn">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set ratio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZRatioLineEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
